AppWidget/app/src/main/res/layout/activity_widget_configure.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (C) 2021 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:padding="16dp">
+
+    <LinearLayout
+        android:id="@+id/grocery_list_container"
```

Comment:
I think the current layout is better aligned with the internal deck.

Reply:
It's causing too much duplication of code, which I worry isn't a good example to set. I think in this case, it's OK if we deviate a little bit from the deck.

Reply:
I'd say it's not too much duplication since each of the list consists of one LinearLayout and three checkboxes.
And I think the appearance looks better with the current layout.

But I extracted some of the common attributes of Checkboxes to style to reduce the duplication.